src/main/kotlin/no/nav/familie/ef/personhendelse/handler/DodsfallHandler.kt

```diff
@@ -14,7 +14,7 @@ class DodsfallHandler(
 ) : PersonhendelseHandler {
     override val type = PersonhendelseType.DØDSFALL
 
-    override fun lagOppgaveInformasjon(personhendelse: Personhendelse): OppgaveInformasjon = UtsettDødsfallOppgave(beskrivelse = "Dødsfall med dødsdato: ${personhendelse.doedsfall.doedsdato.tilNorskDatoformat()}")
+    override fun lagOppgaveInformasjon(personhendelse: Personhendelse): OppgaveInformasjon = UtsettOppgave(beskrivelse = "Dødsfall med dødsdato: ${personhendelse.doedsfall.doedsdato.tilNorskDatoformat()}")
```

Comment:
`UtsattOppgave` istedenfor `UtsettOppgave` ?

Reply:
Enig i denne 👍

Reply:
Eller, skal vi la den hete UtsettOppgave, siden det er en av en sealed class (OppgaveInformasjon) som representerer en handling ?

Reply:
Grunnen til at jeg er enig i kommentaren til Charlie er at jeg synes det ser rart ut å navngi en klasse eller et objekt som om det er en aksjon. Det er vanlig å navngi funkjoner og metoder som aksjoner siden det er nettopp det de er: en aksjon. Dataklasser og objekter er ikke aksjoner i seg selv og det blir derfor rart å navngi dem som om de er det.

Jeg tenker på det som at UtsettOppgave ikke representerer en aksjon men en oppgavetype. When-metoden ser så på hvilken oppgavetype den har med å gjøre og utfører en handling basert på oppgavetypen.

Jeg tenker på det som at data klassen ikke representerer en handling i seg selv. Den 

Jeg ser også i when-funksjonen du referer til at alle oppgavetypene er navngitt som om de er aksjoner, noe jeg egentlig synes er uheldig siden det gjør koden vanskeligere å lese. (I hvertfall for min del) Kunne man ha navngitt `IkkeOpprettOppgave` og `OpprettOppgave` også? De er jo heller ikke aksjoner men kun forskjellige oppgavetyper.

Reply:
Ble en lang tekst men den forklarer hvordan jeg tenker :)

Reply:
Skjønner, og synes jo at det er fint å navngi data classes som du beskriver, men data classes i sealed classes skiller seg litt ut, og brukes noen ganger til å beskrive handlinger på dette viset, litt som en fancy enum :)

Reply:
Så en kan si at OppgaveInformasjon er en sealed class, eller et informasjonsobjekt, som har tre aksjoner, der to av aksjonene har en beskrivelse som vi tar med videre (gir ikke mening å legge til en beskrivelse på IkkeOpprettOppgave)